src/mame/drivers/meritum.cpp

```diff
@@ -81,6 +107,7 @@ class meritum_state : public driver_device
 	bool m_mode;
 	bool m_size_store;
 	bool m_cassette_data;
+	uint8_t m_model; // 1=Meritum I, 2=Meritum II (Meritum I model 2)
```

Comment:
It's generally better to avoid machine type variables like this and just build the different machine types up.